.vscode/launch.json

```diff
@@ -260,6 +260,17 @@
             },
             "cwd": "${workspaceFolder}/test-integration/samples/app-sample-dev/",
             "console": "integratedTerminal"
-        }
+        },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "jhipster openapi-cli",
```

Comment:
Maybe just `openapi` ? We are already in a CLI so that's a bit redundant

Reply:
Yes +1 for just `openapi`, openapi-cli seems weird.

Reply:
Maybe `openapi-client` or `openapi-rest-client` or `openapi-generator`.

Reply:
`openapi-client` is good to me